src/realm.h

```diff
@@ -854,21 +854,6 @@ RLM_API void realm_config_set_scheduler(realm_config_t*, const realm_scheduler_t
  */
 RLM_API void realm_config_set_sync_config(realm_config_t*, realm_sync_config_t*);
```

Comment:
Should this (and a bunch of other methods in the C API) also be removed?

Reply:
Indeed